docs/docs/development.md

```diff
@@ -108,17 +120,18 @@ Afterwards, you should see it listed as a "local extension" when you run
 `jupyter labextension list` and as a normal extension when you run
 `jupyter nbextension list`.
 
-## `perspective-python`
-
-To build the Python library, first configure your project to build Python via
-`yarn setup`, then run:
+As an example, your setup process might look like this:
 
 ```bash
+python -m venv ./venv
+pip install -r python/perspective/requirements-311.txt
+yarn setup # choose python
 yarn build
+yarn setup # choose javascript > jupyterlab
+yarn build
+yarn jlab_link # run this whenever you need to update a local perspective package
```

Comment:
This doesn't actually work due to a bug. `yarn jlab_link` currently _installs_ `perspective-python` in your virtual env, including the python and C++ extension, so further rebuilds in your source repo won't get picked up.